sos/collector/sosnode.py

```diff
@@ -762,6 +764,22 @@ def execute_sos_command(self):
         try:
             path = False
             checksum = False
+            config_file_arg = ''
+            if self.opts.node_config_file:
+                config_file_arg = f'--config-file={self.opts.node_config_file}'
+            elif self.opts.inherit_config_file:
+                if not self.local:
+                    remote_config = f"/tmp/{self.tmpdir.split('/')[-1]}.conf"
+                    self._transport.copy_file_to_remote(
+                        self.opts.config_file,
+                        remote_config)
+                    config_file_arg = f'--config-file={remote_config}'
+                else:
+                    config_file_arg = (
+                        f'--config-file={self.opts.config_file}')
+            self.sos_cmd = (
+                f"{self.sos_cmd} {config_file_arg}"
+                if config_file_arg else self.sos_cmd)
```

Comment:
Isn't it easier just to have

```
if config_file_arg:
    self.sos_cmd = f"{self.sos_cmd} {config_file_arg}"
```

? But I am ok with the current code.

Reply:
That is actually what I had at first. Then the python static analyzer I use complained at me. Of course flake8 didnt like the line length after 😏 

I'm going to use the if syntax and just ignore the analyzer's suggestions since it ends up on multiple lines anyway. 👍🏼